File: UI/eturistickaagencija_admin/lib/models/agencija.dart

import 'package:json_annotation/json_annotation.dart';
part 'agencija.g.dart';
@JsonSerializable()
class Agencija{
  int? id;
  String? adresa;
  String? email;
  String? telefon;

  Agencija(this.id,this.adresa,this.email,this.telefon);

factory Agencija.fromJson(Map<String,dynamic>json)=>_$AgencijaFromJson(json);
Map<String,dynamic>toJson()=>_$AgencijaToJson(this);
}

File: eTuristickaAgencija.Service/Database/Agencija.cs
using System;
using System.Collections.Generic;

namespace eTuristickaAgencija.Service.Database;

public partial class Agencija
{
    public int Id { get; set; }

    public string? Adresa { get; set; }

    public string? Email { get; set; }

    public string? Telefon { get; set; }
}
